MODULE.bazel

```diff
@@ -1,6 +1,137 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_twirl")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
+bazel_dep(name = "rules_java", version = "7.11.1")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_jvm_external", version = "6.4")
+bazel_dep(name = "rules_scala_annex")
+
+rules_scala_annex_version = "bzlmod-migration"
+
+archive_override(
+    module_name = "rules_scala_annex",
+    integrity = "sha256-Gt+S2R53KPhfsK66bhOSXowjb35Ouf24zcT3xFZcyAs=",
+    strip_prefix = "rules_scala-{}".format(rules_scala_annex_version),
+    urls = ["https://github.com/lucidsoftware/rules_scala/archive/refs/heads/{}.zip".format(rules_scala_annex_version)],
+)
+
+bazel_dep(name = "stardoc", version = "0.7.1")
```

Comment:
Can this be moved up with the other `bazel_deps`?

Also I think we can mark this as `dev_dependency`

Reply:
See my comment here:
https://github.com/lucidsoftware/rules_play_routes/pull/57#discussion_r1873547143